content/en/docs/concepts/cluster-administration/node-autoscaling.md

```diff
@@ -0,0 +1,306 @@
+    class na blue_on_white;
+    class k8s,cp white_on_blue;
+{{</ mermaid >}}
+
+### Autoscaler implementations
+
+Cluster autoscaler and Karpenter are the two Node autoscalers currently sponsored by
+[SIG Autoscaling](https://github.com/kubernetes/community/tree/master/sig-autoscaling).
+
+From the perspective of a cluster user, both autoscalers should provide a similar Node autoscaling
+experience. Both will provision new Nodes for unschedulable Pods, and both will consolidate the
+Nodes that are no longer optimally utilized.
+
+Different autoscalers may also provide features outside the Node autoscaling scope described on this
+page, and those additional features may differ between them.
+
+Consult the sections below, and the linked documentation for the individual autoscalers to decide
+which autoscaler fits your use-case better.
+
+#### Cluster Autoscaler
+
+Cluster Autoscaler adds or removes Nodes to pre-configured _Node groups_. Node groups generally map
+to some sort of cloud provider resource group (most commonly a Virtual Machine group). A single
+instance of Cluster Autoscaler can simultaneously manage multiple Node groups. When provisioning,
+Cluster Autoscaler will add Nodes to the group that best fits the requests of pending Pods. When
+consolidating, Cluster Autoscaler always selects specific Nodes to remove as opposed to just
+resizing the underlying cloud provider resource group.
+
+Additional context:
+
+* Documentation overview:
+  https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/README.md
+* Supported cloud providers:
+  https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/README.md#faqdocumentation
+* Cluster Autoscaler FAQ:
+  https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md
+* Contact: https://github.com/kubernetes/community/tree/master/sig-autoscaling#contact
+
+#### Karpenter
+
+Karpenter auto-provisions Nodes based on [NodePool](https://karpenter.sh/docs/concepts/nodepools/)
+configuration provided by the cluster operator. Karpenter handles all aspects of node lifecycle, not
+just autoscaling. It works directly with individual cloud provider resources (most commonly
+individual Virtual Machines), and doesn't rely on cloud provider resource groups.
+
+Additional context:
+
+* Documentation: https://karpenter.sh/
+* Supported cloud providers:
+  https://github.com/kubernetes-sigs/karpenter?tab=readme-ov-file#supported-cloudproviders
+* Karpenter FAQ: https://karpenter.sh/docs/faq/
+* Contact:
+  https://github.com/kubernetes-sigs/karpenter#community-discussion-contribution-and-support
+
+#### Implementation comparison
+
+Main differences between Cluster Autoscaler and Karpenter:
+
+* Cluster Autoscaler provides features related to just Node autoscaling. Karpenter has a wider
+  scope, and also provides features intended for managing Node lifecycle altogether (for example
+  grouping Nodes into pools, handling Node upgrades via disruption).
```

Comment:
Not sure how much value there is talking about "grouping nodes into pools". If I were to call out two features to represent the lifecycle behavior, I'd call out rolling nodes when they've reached a certain lifetime (expiry) and auto-upgrade (which you've already called out)